assets/caniuse.css

```diff
@@ -16,6 +16,30 @@ button.caniuse-cell {
   position: relative;
 }
 
+/* handle case when printing */
+@media print {
+  .caniuse-cell.y::before {
+    background: #8bc34a;
```

Comment:
`background` doesn't show up in print mode, right? So maybe we should remove it.  
```suggestion

```